src/replica/replica.h

```diff
@@ -113,6 +114,7 @@ class replica : public serverlet<replica>, public ref_counter, public replica_ba
                          gpid gpid,
                          const app_info &app,
                          bool restore_if_necessary,
+                         bool duplicate_if_necessary,
```

Comment:
`duplicate_if_necessary = true` means it is duplication follower, and `duplicate_if_necessary = false` means it is duplication master or not in duplication. How about rename it as `is_duplication_follower`?